docs/home.rst

```diff
@@ -96,6 +96,13 @@ OpenVINO 2023.0
 
       Optimize generation of the graph model with PyTorch 2.0 torch.compile() backend
 
+   .. grid-item-card:: Generative AI optimization and deployment
+      :link: gen_ai_guide
+      :link-alt: gen ai
+      :link-type: doc
+
+      Generative AI optimization and deployment
+
```

Comment:
We should consider removing one of the existing tiles, as too many of them may defeat the purpose of exposing highly important information this way. Maybe "Run Inference" is not necessary anymore, as it is well exposed in the menu, unlike in the 22.3 version.

Reply:
you can reshuffle them afterward. But Gen AI doc is something we would like to give more visibility